### ISSUE TITLE
parse_date on date before 1970

### ISSUE THREAD

**Issue by mike-vogel** [gaborcsardi/parsedate#13]
Is parse_date supposed to handle dates before 1970 in MM-DD-YYYY or  MMM DD YYYY format?

**After 1970 works:**
> parse_date("04-15-1971")
[1] "1971-04-15 UTC"
**Before 1970 in YYYY-MM-DD works:**
> parse_date("1969-04-15")
[1] "1969-04-15 UTC"
**After 1970 in MMM DD YYYY works:**
> parse_date("apr 15 1970")
[1] "1970-04-15 UTC"
**Before 1970 in MM-DD-YYYY doesn't work:**
> parse_date("04-15-1969")
[1] "2015-01-04 UTC"
**Before 1970 in MMM DD YYYY doesn't work:**
> parse_date("april 15 1969")
[1] "2016-04-15 UTC"

**Comment by gaborcsardi** [comment]
Yes, I can confirm this. The reason is that we are using the git data parser for non-iso dates, and that parser does not anticipate any old dates, apparently.

Anyway, I'll try to fix this soon.
